Mathlib/Topology/Separation.lean

```diff
@@ -2286,51 +2287,91 @@ instance (priority := 100) NormalSpace.of_compactSpace_r1Space [CompactSpace X]
     NormalSpace X where
   normal _s _t hs ht := .of_isCompact_isCompact_isClosed hs.isCompact ht.isCompact ht
 
-/-- A regular topological space with second countable topology is a normal space.
+/-- Lemma used to prove a regular topological space with Lindelöf topology is a normal space,
+and (todo) a perfectly normal space is a completely normal space. -/
```

Comment:
Can you also add this todo to the beginning of the file, where it will be easier to spot?